src/software/simulation/standalone_simulator.h

```diff
@@ -39,6 +46,49 @@ class StandaloneSimulator
      */
     void setupInitialSimulationState();
 
+    /**
+     * Starts the simulation. If the simulator is already running, this
+     * function does nothing.
+     */
+    void startSimulation();
+
+    /**
+     * Stops the simulation. If the simulator is already stopped, this
+     * function does nothing.
+     */
+    void stopSimulation();
+
+    /**
+     * Sets the slow motion multiplier for the simulation. Larger values
+     * cause the simulation to run in slow motion. For example, a value
+     * of 2.0 causes the simulation to run 2x slower.
+     *
+     * @pre value is >= 1.0
+     *
+     * @param multiplier The slow motion multiplier
+     */
+    void setSlowMotionMultiplier(double multiplier);
+
+    /**
+     * Resets the slow motion multiplier value to let the simulation
+     * run in real-time speed.
+     */
+    void resetSlowMotionMultiplier();
+
+    /**
+     * Sets the state of the ball in the simulation. No more than 1 ball may exist
+     * in the simulation at a time. If a ball does not already exist, a ball
+     * is added with the given state. If a ball already exists, it's state is set to the
+     * given state.
+     *
+     * @param ball_state The new ball state
```

Comment:
The @param is just called `state` below.

Reply:
good catch, fixed